limitador-server/src/main.rs

```diff
@@ -14,6 +14,7 @@ use crate::config::{
 use crate::envoy_rls::server::{run_envoy_rls_server, RateLimitHeaders};
 use crate::http_api::server::run_http_server;
 use crate::metrics::MetricsLayer;
+use ::metrics::histogram;
```

Comment:
neat: `crate` prefix to stick to the pattern...

Reply:
I think this is `::metrics::histogram` to match the upstream metrics crate, and not the local metrics module (used on the line above).. should probably rename the metrics module (containing the tracing metrics layer) to be clearer

Reply:
Oh yeah... duh! my bad. Guess the `::` threw me off, `historgram` is a macro 🤦 
Ignore me!